### PR TITLE
Add build option to explicitly specify BLAS/LAPACK

### PR DESCRIPTION
**Pull request by jeanlucf22**
Will use environment variables BLAS_LIBRARIES and
LAPACK_LIBRARIES if defined to specify where those libraries are.

---
This change is [Reviewable](https://reviewable.io/reviews/lanl/bml/321)

### CODE REVIEW

**Review by suemni — APPROVED** [review]
Sounds good!

File: CMakeLists.txt [hunk 1]
```diff
@@ -315,13 +315,19 @@ elseif(BLAS_VENDOR STREQUAL "Auto")
   set(LAPACK_FOUND TRUE)
   message(STATUS "I will skip BLAS/LAPACK detection")
 elseif(BLAS_VENDOR STREQUAL "")
-  message(STATUS "I will rely on the default paths to find BLAS/LAPACK")
+  message(STATUS "I will rely on the default paths or enironment variables to find BLAS/LAPACK")
```
Comment (nicolasbock) [inline CMakeLists.txt]:
```suggestion
  message(STATUS "I will rely on the default paths or environment variables to find BLAS/LAPACK")
```

**Review by nicolasbock — CHANGES_REQUESTED** [review]
Thanks, looks great! Just one small spelling suggestion.

**Review by nicolasbock — APPROVED** [review]
Thanks for fixing!

**Pull request merged**